src/ClusterManager/framework.py

```diff
@@ -318,7 +318,7 @@ def gen_containers(job, role):
     if job.init_image is None:
         cmd = [
             "sh", "-c",
-            "printenv DLWS_LAUNCH_CMD > /job_command.sh ; mkdir -p /dlts-runtime/status ; touch /dlts-runtime/status/READY ; sh /job_command.sh"
+            "printenv DLWS_LAUNCH_CMD > /job_command.sh ; mkdir -p /dlts-runtime/status ; touch /dlts-runtime/status/READY ; bash -x /job_command.sh"
```

Comment:
Should we change this for distributed and regular job as well?

Reply:
solved